### PR TITLE
[NUI] Add TCs for Event

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
Add TCs for Event

### API Changes ###

 - ACR:

### CODE REVIEW

File: test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Events/TSGestureDetector.cs [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+           
+            GestureDetector newTwo = new GestureDetector(GestureDetector.getCPtr(ret).Handle, true);
+            Assert.IsNotNull(newTwo, "should be not null");
+            Assert.IsInstanceOf<GestureDetector>(newTwo, "should be an instance of testing target class!");
+
+            ret.Dispose();
+            newOne.Dispose();
+            newTwo.Dispose();
+			tlog.Debug(tag, $"GestureDetectorConstructor END (OK)");
+			Assert.Pass("GestureDetectorConstructor");
```
Comment (zg2nets) [inline test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Events/TSGestureDetector.cs:57]: plz fix align

File: test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Events/TSGestureDetector.cs [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+            Assert.IsInstanceOf<GestureDetector>(ret, "should be an instance of testing target class!");
+
+            View view = new View();
+            ret.Attach(view);
+
+            ret.Detach(view);
+            ret.Dispose();
+            
+			tlog.Debug(tag, $"GestureDetectorAttach END (OK)");
+			Assert.Pass("GestureDetectorAttach");
```
Comment (zg2nets) [inline test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Events/TSGestureDetector.cs:82]: plz fix align

File: test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Events/TSGestureDetector.cs [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+        [Test]
+        [Category("P1")]
+        [Description("GestureDetector Detach")]
+        [Property("SPEC", "Tizen.NUI.GestureDetector.Detach M")]
+        [Property("SPEC_URL", "-")]
+        [Property("CRITERIA", "MR")]
+        [Property("AUTHOR", "guowei.wang@samsung.com")]
+        public void GestureDetectorDetach()
+        {
+			tlog.Debug(tag, $"GestureDetectorDetach START");
```
Comment (zg2nets) [inline test/Tizen.NUI.Tests/Tizen.NUI.TCT/testcase/public/Events/TSGestureDetector.cs:94]: plz fix align

**Review by zg2nets — COMMENTED** [review]
plz fix align

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by huayongxu — APPROVED** [review]
lgtm

**Review by wb0716 — APPROVED** [review]
good for me

**Review by elishateng — APPROVED** [review]
LGTM

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**